accountServices.py

```diff
@@ -11,8 +11,8 @@ def __init__(self, config):
         self.config = config
         self.words = config["WORDS"]
         self.logger = logging.getLogger("AcctServices")
-        self.kadmin = kadmin.KAdmin(config["krb5"]["aprinc"],
-                                    config["krb5"]["atab"])
+        self.kadm = kadmin.init_with_keytab(config["krb5"]["aprinc"],
+                                            config["krb5"]["atab"])
```

Comment:
s/atab/keytab/

But really it shouldn't be set at all.  Its pretty standard to just use the system keytab for applications that are binding with machine level access.